cvm/docs/README.md

```diff
@@ -32,44 +32,75 @@ The Calibration and Quantization pass is achieved in class MRT.
-| Pooling       | :heavy_check_mark: | broadcast_greater | :x:                |
-| broadcast_mul | :heavy_check_mark: | Concat            | :heavy_check_mark: |
-| broadcast_add | :heavy_check_mark: | sum               | :heavy_check_mark: |
-| BatchNorm     | :x:                | ceil              | :x:                |
-| Flatten       | :heavy_check_mark: | round             | :x:                |
-| floor         | :x:                | fix               | :x:                |
-| Cast          | :x:                | clip              | :heavy_check_mark: |
-| Reshape       | :heavy_check_mark: | _minimum          | :x:                |
-| Custom        | :x:                | _maximum          | :heavy_check_mark: |
-| max           | :heavy_check_mark: | min               | :x:                |
-| argmax        | :x:                | argmin            | :x:                |
-| abs           | :x:                | elemwise_add      | :heavy_check_mark: |
-| elemwise_sub  | :heavy_check_mark: | Dropout           | :heavy_check_mark: |
-| _arange       | :heavy_check_mark: | tile              | :heavy_check_mark: |
-| negative      | :heavy_check_mark: | SwapAxis          | :x:                |
-| _plus_scalar  | :x:                | zeros_like        | :x:                |
-| ones_like     | :x:                | _greater_scalar   | :x:                |
-| where         | :x:                | squeeze           | :heavy_check_mark: |
+The list operators have already been considered by MRT developers, which operators over the list is not allowed in quantization. Split the model for unsupported operators with disable-quantization attributes or contact the MRT developers in the github for operators needing quantization.
+
+##### Transformer
+
+| Operator     | Supported          | Operator    | Supported          |
+| ------------ | ------------------ | ----------- | ------------------ |
+| SliceAxis    | :heavy_check_mark: | SwapAxis    | :heavy_check_mark: |
+| Slice        | :heavy_check_mark: | Flatten     | :heavy_check_mark: |
+| SliceLike    | :heavy_check_mark: | Concat      | :heavy_check_mark: |
+| Transpose    | :heavy_check_mark: | where       | :heavy_check_mark: |
+| repeat       | :heavy_check_mark: | expand_dims | :heavy_check_mark: |
+| SliceChannel | :heavy_check_mark: | tile        | :heavy_check_mark: |
+| squeeze      | :heavy_check_mark: | Reshape     | :heavy_check_mark: |
+| clip         | :heavy_check_mark: | Embedding   | :heavy_check_mark: |
+
+##### NN
 
+| Operator       | Supported          | Operator | Supported          |
+| -------------- | ------------------ | -------- | ------------------ |
+| Convolution    | :heavy_check_mark: | Pad      | :heavy_check_mark: |
+| FullyConnected | :heavy_check_mark: | relu     | :heavy_check_mark: |
+| LeakyReLU      | :heavy_check_mark: | Pooling  | :heavy_check_mark: |
+| UpSampling     | :x:                | softmax  | :heavy_check_mark: |
```

Comment:
operator upsampling  not support?

Reply:
Currently not.

Reply:
The user has to split 'upsampling' into other part. because neither quantization has been registered or other equivalent transitions exits. MRT would raise runtime error while processing this op.

Reply:
Currently this op will be processed only in the compile stage.